.tekton/integration-tests/coo-e2e-test-fbc-ocp-ci-job-trigger-pipeline.yaml

```
@@ -0,0 +1,168 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  annotations:
+    pipelinesascode.tekton.dev/task: "[coo-e2e-tests]"
+  name: coo-e2e-test-fbc
+spec:
+  description: |
+    This pipeline automates the process of running e2e tests for Cluster Observability Operator. The pipeline fetches the FBC fragment image and
+    passes it as a parameter to the OpenShift CI job using Gangway API.
+  params:
+    - name: SNAPSHOT
+      description: 'The JSON string representing the snapshot of the application under test.'
+      default: '{"components": [{"name":"test-app", "containerImage": "quay.io/example/repo:latest"}]}'
+      type: string
+    - name: test-name
+      description: 'The name of the test corresponding to a defined Konflux integration test.'
+      default: 'coo-e2e-test-fbc'
+      type: string
+    - name: gangway_api_url
+      description: 'URL for the Gangway API'
+      default: 'https://gangway-ci.apps.ci.l2s4.p1.openshiftapps.com'
+      type: string
+    - name: ocp_ci_job_name
+      description: 'OpenShift CI job name'
+      type: string
+    - name: multistage_param_override_key
+      description: 'The key name for the multistage parameter override (e.g., MULTISTAGE_PARAM_OVERRIDE_COO_INDEX_IMAGE)'
+      type: string
+  tasks:
+    - name: parse-metadata
+      taskRef:
+        resolver: git
+        params:
+          - name: url
+            value: https://github.com/konflux-ci/integration-examples
```

Should we really use this repo? The readme says 
> Examples for Integration Test definitions that can be used for testing and experimenting with the Konflux Integration Service, including Tekton tasks and pipelines as well as Custom resource definitions of Konflux APIs.

Is there maybe a better source for these tasks?

---

https://github.com/konflux-ci/tekton-integration-catalog/blob/main/tasks/test-metadata/0.2/test-metadata.yaml 
0.2 version of the task can't work for me for miss results vairable `component-container-imag`
So I changed back to use task in repo integration-examples

---

Perhaps you could add a todo comment, rather than have this PR blocked:

```suggestion
            # Todo: Use a better task definition here instead of the Konflux example
            value: https://github.com/konflux-ci/integration-examples
```

---

What's the issue with the 0.3 version? Maybe it can be fixed..?

---

Thanks @simonbaird , just proposed a fix for 0.3 test-metadata.
https://github.com/konflux-ci/tekton-integration-catalog/pull/206/files 
Not sure when the PR can be merged, will wait for a day or two, or else will add todo comment as you suggested